client/js/components/month/month-week-board.js

```diff
@@ -1,15 +1,20 @@
 /** @jsx React.DOM */
 var React = require('react');
-var MonthDayCard = require('./month-day-card.js');
+var MonthDayCard = require('./month-day-card');
 var _ = require('underscore');
+var Link = require('react-router-component').Link
 
 var MonthWeekBoard = React.createClass({
   render: function () {
     var weekEvents = _.map(this.props.weekEvents, function(day, key) {
```

Comment:
Agreed. I think the final naming convention will be largely informed by what the data is actually going to look like. For now im definitely happy to rename it to whatever is clearer for code reviews. And similar response to your second point, Id like to hold off on modifying the mock data until we have finalized version one of our schema and spoken to the company more about the data we can expect